packages/muelu/src/Operators/MueLu_Maxwell1_def.hpp

```diff
@@ -107,10 +106,20 @@ namespace MueLu {
       // interpret ML list
       newList.sublist("maxwell1: 22list") = *Teuchos::getParametersFromXmlString(MueLu::ML2MueLuParameterTranslator::translate(list,"Maxwell"));
 
+
+      // Hardwiring options to ensure ML compatibility
+      newList.sublist("maxwell1: 22list").set("use kokkos refactor", false);
```

Comment:
Would it make more sense to set that as default instead of hardwiring it?

Reply:
Oh wait, this is the ML translation only.

Reply:
Right now it is hardwired to use an ML style list too...